tests/interfaces/default_qubit_2_integration/test_autograd_default_qubit_2.py

```diff
@@ -223,6 +259,8 @@ def test_tape_no_parameters(self, execute_kwargs, shots, device):
 
         if execute_kwargs["gradient_fn"] == "adjoint":
             pytest.skip("Adjoint differentiation does not yet support probabilities")
+        if shots.has_partitioned_shots:
+            pytest.skip("needs further investigation")
```

Comment:
Is this tracked anywhere? :)

Reply:
fixed. hstack doesn't like nested structures.